IFC_Engine/Compute/UnsupportedUnitsWarning.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+
+namespace BH.Engine.Adapters.IFC
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Throws a warning regarding lack of support for units in property conversion.")]
+        public static void UnsupportedUnitsWarning()
```

Comment:
Maybe have this as a private method instead? 

I know the struggle - this is one of our internal BHoM idiosyncrasies, especially if you need to call this method from different Engine classes. If you only need it in `Compute`, though, I'd make it private: no need to expose it in the UI.

Reply:
I was thinking about it making it private. However, then in case of need for similar method, there is ~100% chance a duplicate will be created (who would look for a private method?). Therefore public, taken how broad the application of this can be...

Reply:
If it doesn't need to be exposed to the UI, why not stick it in the `Objects` folder? Could be public in there and have it's own class as needed and is exempt from most compliance rules barring a few on file name stuff.

Reply:
Hmm, but isn't `Objects` folder meant to contain the actual definitions of objects that can be instantiated? Not sure if putting a static class makes full sense to me?

Reply:
Either way, this is an unresolved BHoM Framework problem which I don't believe is right to resolve by adding a burden to users. In these cases, I personally prefer `private` code duplication to exposing irrelevant methods to the UI, as they get reflected in Wiki, Documentation and API. This code duplication also rightly exposes a "code smell" which correctly highlights a framework issue.

Reply:
I agree with you @alelom, but I do not feel comfortable with neither of the proposed alternatives. Instead, to reduce the noise I removed the `UnsupportedUnitsWarning` method and called the warning directly from within the methods - after a second look this makes most sense to me, particularly taken the warning is only temporary and will rather not be reused in the foreseeable future. How does that sound? 😉

Reply:
A single private method was not an option as the methods sit in 2 separate classes - otherwise I would also opt for such approach. Anyways, thanks for reviewing!